content/ops/contingency-plan.md

```diff
@@ -0,0 +1,122 @@
+* Data loss that causes disruption:
+  * By mistake we delete or corrupt data that is essential to our service, causing us to need to restore an essential system or configuration from backup, and this causes disruption in our service delivery.
+* High-severity security incidents:
+  * A high-severity security incident related to system confidentiality/integrity (such as a root-level compromise) requires us to rebuild components of cloud.gov based on trusted sources.
+  * A high-severity security incident related to system availability (such as a malicious DDOS) requires mitigation.
+* Two or more of these things happen at the same time.
+
+---
+
+## Outline of essential functions and their requirements
+
+These are essential functions of our cloud.gov information systems, listed in order of which would need to be restored first in case of critical problems with more than one function:
+
+* The cloud.gov team is able to communicate with each other. Key components and plans to restore this function if disrupted:
+  * We primarily use 18F Slack to communicate among our geographically distributed team. If Slack is unavailable or expected to be unavailable for more than 30 minutes while we're working on a time-sensitive problem (or 6 hours while doing ordinary work), team leads will reach out to team members via GSA email to coordinate use of GSA Google Apps (Gmail, Hangouts, and Drive) as a temporary replacement.
+  * If neither Slack nor GSA Google Apps are available, team leads will coordinate using the GSA phone numbers of team members.
+* The cloud.gov team is able to diagnose and mitigate issues. Key components and plans to restore this function if disrupted:
+  * We need access to the critical platforms and services that cloud.gov relies on: AWS. If we lose access, team members who have access to the credentials and contacts for these systems should work to restore access.
+  * We need to be able to log into our cloud.gov systems, get information from them, and modify them. If we lose access, team members who have access to the credentials for these systems should work to restore access.
+  * We need access to our logging and notification systems. If we lose access, team members who have access to the credentials for these systems should work to restore access.
+  * We need access to our code and documentation, which means access to the information in our 18F GitHub organization. If we lose access to GitHub, each cloud.gov team member has local copies of some cloud.gov git repositories created as part of their ordinary day-to-day work, which can serve as a temporary resource. The 18F Director of Infrastructure makes systematic backups of 18F GitHub repositories at a daily frequency.
```

Comment:
Did you get confirmation of this? If so, I'll add a note in the Handbook and link there.